src/runtime/composables/turnstile.ts

```diff
@@ -0,0 +1,19 @@
+import { useScript } from "#imports"
+
+declare global {
+    interface Window {
+        turnstile: Turnstile.Turnstile
+    }
+}
+
+export function useScriptCloudflareTurnstile() {
+    return useScript({
```

Comment:
You need to pass the generic in `useScript<Turnstile.Turnstile>({ ... })` for the types to work